chef/cookbooks/ceph/recipes/radosgw_civetweb.rb

```diff
@@ -0,0 +1,79 @@
+# This recipe doesn't do anything other than deal with SSL keys and certificates.
+# Everything else for civetweb setup is in conf.rb.  It could arguably be merged
+# into radosgw.rb; it only exists as a separate file because it used to be
+# radosgw_apache2.rb
+
+if node["ceph"]["radosgw"]["ssl"]["enabled"]
+  certfile      = node["ceph"]["radosgw"]["ssl"]["certfile"]
+  keyfile       = node["ceph"]["radosgw"]["ssl"]["keyfile"]
+  pemfile       = node["ceph"]["radosgw"]["ssl"]["pemfile"]
+  if  node["ceph"]["radosgw"]["ssl"]["generate_certs"]
+    package "openssl"
+    ruby_block "generate_certs for radosgw" do
+        block do
```

Comment:
Style/IndentationWidth: Use 2 (not 4) spaces for indentation. (https://github.com/bbatsov/ruby-style-guide#spaces-indentation)